libc-top-half/musl/src/thread/pthread_cond_timedwait.c

```diff
@@ -156,7 +173,7 @@ int __pthread_cond_timedwait(pthread_cond_t *restrict c, pthread_mutex_t *restri
 		if (val>0) a_cas(&m->_m_lock, val, val|0x80000000);
 		unlock_requeue(&node.prev->barrier, &m->_m_lock, m->_m_type & (8|128));
 	} else if (!(m->_m_type & 8)) {
-		a_dec(&m->_m_waiters);		
+		a_dec(&m->_m_waiters);
```

Comment:
revert these 3 whitespace-only changes?

Reply:
I guess that's to keep the source as close to musl as possible? I've reverted that (and two similar) changes.